pkgs/development/tools/build-managers/mill/default.nix

```diff
@@ -1,26 +1,33 @@
-{ stdenv, lib, fetchurl, makeWrapper, jre, gnused }:
+{ stdenv, fetchurl, jre }:
  stdenv.mkDerivation rec {
   name = "mill-${version}";
   version = "0.3.5";
-    src = fetchurl {
-     url = "https://github.com/lihaoyi/mill/releases/download/${version}/${version}";
-     sha256 = "19ka81f6vjr85gd8cadn0fv0i0qcdspx2skslfksklxdxs2gasf8";
+
+  src = fetchurl {
+    url = "https://github.com/lihaoyi/mill/releases/download/${version}/${version}";
+    sha256 = "19ka81f6vjr85gd8cadn0fv0i0qcdspx2skslfksklxdxs2gasf8";
   };
-   propagatedBuildInputs = [ jre ] ;
-  buildInputs = [ makeWrapper gnused ] ;
-   phases = "installPhase";
-   installPhase = ''
+
+  propagatedBuildInputs = [ jre ] ;
+
+  phases = "installPhase";
```

Comment:
This is not a good way. You are throwing away the standard `unpackPhase` and `fixupPhase`.
I think its fine to instead let the standard phases fail here, they should not terminate the build. You can ask to skip some of the phases by setting
```nix
stdenv.mkDerivation {
# ...
dontConfigure = true;
dontBuild = true;
# ...
}
```
Not that `dontConfigure` doesn't work at the moment, but it's worth to state the intent.

Reply:
Thanks again for helping out on this. I've added the flags you mention, I also override the unpackPhase, so there are no errors occuring during the build. Is this an ok way of doing this? Without it, my build did actually stall locally.

I've also removed the sed patching, it was not necessary it seems.